docs/content/guide/component.ngdoc

```diff
@@ -282,7 +282,7 @@ it upwards to the heroList component, which updates the original data.
 </file>
 <file name="editableField.html">
   <span ng-switch="$ctrl.editMode">
-    <input ng-switch-when="true" type="text" ng-model="$ctrl.fieldValue">
+    <input ng-switch-when="true" type="{{::$ctrl.fieldType}}" ng-model="$ctrl.fieldValue">
```

Comment:
Hmm, is not it the best practice for performance?
I mean in the example the `fieldType` is not changed dynamically so it makes it more transparent, what is wanted to be achieved.
Finally, for the `::` I can add a note, with link to read about it.